index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script
+      type="text/javascript"
+      src="https://cdn.jsdelivr.net/npm/lozad/dist/lozad.min.js"
+    ></script>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav class="navigation">
+          <div class="menu">
+            <div class="header">
+              <div id="icon">
+                <a>
+                  <img
+                    class="light"
+                    src="https://a.amz.mshcdn.com/assets/header_logo.v2.us-6653972cc831bd09732d26dc9da67bd6be0c4d2b28757f116791336ccf2146ec.png"
+                  />
+                  <img
+                    class="dark"
+                    src="https://a.amz.mshcdn.com/assets/header_logo.v2.us.dark-dd0a18bfb3d211980944b582cfc03c24db96b9069f2a3234fb6cf064c72f58aa.png"
+                  />
+                </a>
+              </div>
+            </div>
+            <ul class="menu-links"></ul>
+          </div>
+
+          <div class="social">
+            <ul class="social-links"></ul>
+          </div>
+
+          <div class="hamburger">
+            <div class="line1"></div>
```

Comment:
This can be just one class I think instead of 3 😅

Reply:
I think that the way i use it it needs to be like that, but i will check it.

Reply:
@oscarpolanco i need the three classes 'cause each class have a different styling